scio-avro/src/main/scala/com/spotify/scio/avro/AvroSCollection.scala

```diff
@@ -37,18 +40,31 @@ final class AvroSCollection[T](@transient val self: SCollection[T]) extends Seri
    * @param schema must be not null if `T` is of type
    *               [[org.apache.avro.generic.GenericRecord GenericRecord]].
    */
+  // scalastyle:off parameter.number
   def saveAsAvroFile(path: String,
                      numShards: Int = AvroIO.WriteParam.DefaultNumShards,
                      schema: Schema = null,
```

Comment:
Unless I'm missing something Schema should never be null here.

Reply:
That's correct. I will fix that.